cfme/cloud/availability_zone.py

```diff
@@ -35,6 +34,7 @@ class AvailabilityZoneDetailsToolBar(View):
 class AvailabilityZoneEntities(BaseEntitiesView):
     """View containing the widgets for the main content pane"""
     table = Table("//div[@id='gtl_div']//table")
+    paginator = PaginationPane()
```

Comment:
Since this inherits BaseEntitiesView, paginator should be available through BaseEntitiesView.entities, which is an EntitiesConditionalView that contains the paginator widget.

Starting review, so there's likely places where this is directly called below, and removing it would force a change to those call chains. 

`entities` here: https://github.com/ManageIQ/integration_tests/blob/master/widgetastic_manageiq/__init__.py#L3142

and `paginator` here: https://github.com/ManageIQ/integration_tests/blob/master/widgetastic_manageiq/__init__.py#L2940